roles/logging-config/templates/etc/logstash-forwarder.d/template.conf

```diff
@@ -0,0 +1,23 @@
+{
+  "files": [
+    {% if logdata -%}
+      {% for item in logdata %}
+      {
+      "paths": [
+	{% for path in item.paths %}
+	"{{ path }}"{% if not loop.last %},{% endif %}
+
+	{% endfor %}
+      ],
+      "fields": {
+	{% for key, value in item.fields.items() %}
+	"{{ key }}": "{{ value }}"{% if not loop.last %},{% endif %}
+
+	{% endfor %}
+      }
+      }{% if not loop.last %},{% endif %}
```

Comment:
This is the same as it is being done in 1.2.x, but I will check into modifying the template if this would suffice instead.